src/package.json

```diff
@@ -37,6 +37,7 @@
     "esbuild": "^0.15.10",
     "helmet": "^6.0.0",
     "svgo": "^2.8.0",
```

Comment:
Already discussed in slack, going to see how difficult https://github.com/Psifi-Solutions/csrf-sync would be to add.

Reply:
I think with csrf-sync, we will now have to manually generate token per session or per request. 
The backend will generate via the `generateToken` method and the frontend will have to take that and add it to the request header in order for the request to go through..

Reply:
@rhelmer with doublecsrf I believe the same flow is still needed as `csrf-sync`... As in every request (POST PUT PATCH DELETE etc) needs the token in the request header from `generateToken`.

Reply:
But I think this is what's intended.. When the frontend sends requests via POST etc, we will have to be mindful to set the header `x-csrf-token`